### PR TITLE
Added Java Project Structure. Code need to be checked and structure also accepted.

### PR DESCRIPTION
**Pull request by carlillous**
Code is WIP i guess. what would you change etc etc @adam-prochazka.

By the way is all @carlos-esplaa fault if something is wrong. I think i managed to win vs Git this time hehe . Was a little bit hard not gonna lie (no it wasn't).

### CODE REVIEW

**Comment by carlillous** [comment]
Also artifacts must be renamed i think.

File: java/Crawler/src/main/java/GutenbergScraper.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.jsoup.select.Elements;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+// Descargar cada minuto y antes del nombre del libro que se descargue y ordene por fecha.
```
Comment (adam-prochazka) [inline java/Crawler/src/main/java/GutenbergScraper.java:11]: Nah nah naaah

File: java/Crawler/src/main/java/GutenbergScraper.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+            if (downloadLink != null) {
+                String downloadUrl = "https://www.gutenberg.org" + downloadLink.attr("href");
+                String bookTitle = doc.title();
+                String fileName = bookTitle + ".txt";
+                if (!fileExists(fileName)) {
+                    String bookContent = Jsoup.connect(downloadUrl).ignoreContentType(true).execute().body();
+                    saveToFile(fileName, bookContent);
+                } else {
+                    System.out.println("El libro ya existe: " + fileName);
```
Comment (adam-prochazka) [inline java/Crawler/src/main/java/GutenbergScraper.java:43]: Naaaah nah nah

File: java/Crawler/src/main/java/GutenbergScraper.java [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+    private boolean fileExists(String fileName) {
+        File file = new File(fileName);
+        return file.exists();
+    }
+
+    private void saveToFile(String fileName, String content) {
+        try (BufferedWriter writer = new BufferedWriter(new FileWriter(fileName))) {
+            writer.write(content);
+            System.out.println("Libro guardado: " + fileName);
```
Comment (adam-prochazka) [inline java/Crawler/src/main/java/GutenbergScraper.java:59]: Nah naah nnaaaahhh

**Review by adam-prochazka — CHANGES_REQUESTED** [review]
Looks good except for some minor issues in the crawler. Also, can we call it "Crawler" and not "Scraper"? Just asking :)

File: java/Crawler/src/main/java/GutenbergScraper.java (hunk 3 above)
Comment (carlillous) [reply java/Crawler/src/main/java/GutenbergScraper.java:59]: sensei i told you that the code was written by carlos the second it is not my fault........

and i think it is not even finished yet soooooooooooo im deleting it from the pull request

**Comment by adam-prochazka** [comment]
That is the easy part hahahahah

**Pull request merged**